coverage.xml

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" ?>
```

Comment:
We might need to add this to .gitignore, like in Trainer: https://github.com/kubeflow/trainer/blob/master/.gitignore#L23-L24

Reply:
done